instrumentation-genai/opentelemetry-instrumentation-openai-v2/src/opentelemetry/instrumentation/openai_v2/__init__.py

```diff
@@ -52,7 +52,7 @@
 from opentelemetry.semconv.schemas import Schemas
```

Comment:
- Can you update changelog as well https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation-genai/opentelemetry-instrumentation-openai-v2/CHANGELOG.md
- In README, seems the link [OpenTelemetry OpenAI Instrumentation](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/openai/openai.html) is not working any more.
- Can we add a list of instrumented APIs for OpenAI in readme as well?

Reply:
sure!

Reply:
@gyliu513, added changelog and adjusted readme

Reply:
reverted changes, as i see @lmolkova having the same changes specified in this [PR](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2988)